.ci.yaml

```diff
@@ -1215,6 +1215,42 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Linux android_java11_dependency_smoke_tests
+    recipe: devicelab/devicelab_drone
+    presubmit: false
```

Comment:
Is this presubmit false because this test is slow?

Reply:
Yes, that was my intention